horizontalintegration/xmc-hztl-foundation-2024-q1
# Feature/fis 330 css cleanup

## What does this PR do and why?

This PR removes some CSS files that were inherited from an old SXA project, refactored the remaining, required CSS files to use the proper Tailwind classes via the @apply statement, and removes some stub components that are generated by the JSS scaffolding process.

## Screenshots or screen recordings

N/A

## How to set up and validate locally

1. Check out the branch _feature/FIS-330-css-cleanup_.
2. From a command prompt, navigate to _./src/hztl-foundation_ and execute `npm run storybook`.
3. Once Storybook loads, carefully audit each component to ensure it renders correctly.
4. From a second command prompt, navigate to _./src/hztl-foundation_ and execute `npm start:connected`.
5. From a browser window, navigate to http://localhost:3000, and carefully audit each page to ensure it renders correctly.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
